labgrid/util/managedfile.py

```diff
@@ -67,17 +67,19 @@ def sync_to_resource(self, symlink=None):
                     self.local_path,
                     f"{self.rpath}{os.path.basename(self.local_path)}"
                 )
-
-            if symlink is not None:
-                self.logger.info("Linking")
-                try:
-                    conn.run_check(f"test ! -e {symlink} -o -L {symlink}")
-                except ExecutionError:
-                    raise ManagedFileError(f"Path {symlink} exists but is not a symlink.")
-                # use short options to be compatible with busybox
-                # --symbolic --force --no-dereference
-                conn.run_check(f"ln -sfn {self.rpath}{os.path.basename(self.local_path)} {symlink}")
-
+        else:
+            conn = sshmanager.open("localhost")
```

Comment:
This looks broken, why do we need to open an SSH connection when operating on a local machine? This breaks the sandboxed tests on nixpkgs since we now open SSH connections for tests that never did this before.

Reply:
Hm okay, I thought it would be a good idea to not duplicate the symlinking logic locally in Python, but you're right, it would actually be better.